integration-tests/bats/sql.bats

```diff
@@ -1694,3 +1694,22 @@ SQL
 get_head_commit() {
     dolt log -n 1 | grep -m 1 commit | cut -c 15-46
 }
+
+@test "sql: query return format check" {
```

Comment:
I would also like to see an expect file test running through different line terminators